### ISSUE TITLE
NoClassDefFoundError: android.view.accessibility.AccessibilityNodeInfo$CollectionInfo on Samsung Galaxy S3, Android 4.3

### ISSUE THREAD

**Issue by hartmannj** [flutter/flutter#28762]
Another Google Pre-Launch report with a crash on Samsung Galaxy S3, Android 4.3:

```
java.lang.NoClassDefFoundError: android.view.accessibility.AccessibilityNodeInfo$CollectionInfo
	at io.flutter.view.AccessibilityBridge.createAccessibilityNodeInfo(AccessibilityBridge.java:329)
	at android.view.AccessibilityInteractionController$AccessibilityNodePrefetcher.prefetchDescendantsOfVirtualNode(AccessibilityInteractionController.java:907)
	at android.view.AccessibilityInteractionController$AccessibilityNodePrefetcher.prefetchDescendantsOfVirtualNode(AccessibilityInteractionController.java:917)
	at android.view.AccessibilityInteractionController$AccessibilityNodePrefetcher.prefetchDescendantsOfVirtualNode(AccessibilityInteractionController.java:917)
	at android.view.AccessibilityInteractionController$AccessibilityNodePrefetcher.prefetchDescendantsOfVirtualNode(AccessibilityInteractionController.java:917)
	at android.view.AccessibilityInteractionController$AccessibilityNodePrefetcher.prefetchDescendantsOfVirtualNode(AccessibilityInteractionController.java:917)
	at android.view.AccessibilityInteractionController$AccessibilityNodePrefetcher.prefetchDescendantsOfRealNode(AccessibilityInteractionController.java:828)
	at android.view.AccessibilityInteractionController$AccessibilityNodePrefetcher.prefetchDescendantsOfRealNode(AccessibilityInteractionController.java:824)
	at android.view.AccessibilityInteractionController$AccessibilityNodePrefetcher.prefetchDescendantsOfRealNode(AccessibilityInteractionController.java:824)
	at android.view.AccessibilityInteractionController$AccessibilityNodePrefetcher.prefetchAccessibilityNodeInfos(AccessibilityInteractionController.java:710)
	at android.view.AccessibilityInteractionController.findAccessibilityNodeInfoByAccessibilityIdUiThread(AccessibilityInteractionController.java:147)
	at android.view.AccessibilityInteractionController.access$300(AccessibilityInteractionController.java:49)
	at android.view.AccessibilityInteractionController$PrivateHandler.handleMessage(AccessibilityInteractionController.java:961)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:176)
	at android.app.ActivityThread.main(ActivityThread.java:5419)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:525)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1046)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:862)
	at dalvik.system.NativeStart.main(Native Method)
```
And version:
```
Flutter 1.3.0 • channel dev • https://github.com/flutter/flutter.git
Framework • revision 3054a93579 • 2019-02-27 20:22:28 -0500
Engine • revision 3e4e6f5c54
Tools • Dart 2.2.0 (build 2.2.0-dev.2.1 0f9a5198bd)
```

**Comment by jonahwilliams** [comment]
We're missing an API guard on the usage of that accessibility class

**Comment by kw2019ltd** [comment]
same issue in my side NoClassDefFoundError: android.view.accessibility.AccessibilityNodeInfo$CollectionInfo

**Comment by jonahwilliams** [comment]
Fixed via https://github.com/flutter/engine/pull/8061 and rolled into framework via 0d3c150cfc1b3aa86e953afb4b82d232f498dac7

**Comment by yudaprama** [comment]
@jonahwilliams Which channel this fix is available?

**Comment by godlessturtle** [comment]
same problem, is there any solution?

**Comment by HPH92** [comment]
@jonahwilliams how do we get the branch with the fix?

**Comment by zukinaru** [comment]
same problem, any solution ?

**Comment by foxanna** [comment]
I'm still facing this issue with my fresh Android app release.

```
[✓] Flutter (Channel stable, v1.2.1, on Mac OS X 10.14.5 18F132, locale en-UA)
    • Flutter version 1.2.1 at ...
    • Framework revision 8661d8aecd (3 months ago), 2019-02-14 19:19:53 -0800
    • Engine revision 3757390fa4
    • Dart version 2.1.2 (build 2.1.2-dev.0.0 0a7dcf17eb)

[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at ...
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • ANDROID_HOME = ...
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)
    • All Android licenses accepted.

[!] iOS toolchain - develop for iOS devices (Xcode 10.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.2.1, Build version 10E1001
    ✗ ios-deploy not installed. To install:
        brew install ios-deploy
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without resolving iOS dependencies with CocoaPods, plugins will not work on iOS.
        For more info, see https://flutter.io/platform-plugins
      To install:
        brew install cocoapods
        pod setup

[✓] Android Studio (version 3.4)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 35.3.1
    • Dart plugin version 183.6270
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1343-b01)

[✓] VS Code (version 1.34.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.0.2

[✓] Connected device (1 available)
    • Android SDK built for x86 • emulator-5554                        • android-x86 • Android 9 (API 28) (emulator)

! Doctor found issues in 1 category.
```

**Comment by foxanna** [comment]
The issue is gone from my Crashlytics reports after I update Flutter and create a new app release.

```
[✓] Flutter (Channel stable, v1.5.4-hotfix.2, on Mac OS X 10.14.5 18F132, locale en-UA)
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] iOS toolchain - develop for iOS devices (Xcode 10.2.1)
[✓] Android Studio (version 3.4)
[✓] Android Studio
[✓] VS Code (version 1.34.0)
[✓] Connected device (2 available)
```
